api/app/Providers/RouteServiceProvider.php

```diff
@@ -57,7 +57,14 @@ public function boot()
     protected function configureRateLimiting()
     {
         RateLimiter::for('api', function (Request $request) {
```

Comment:
Our auth end points are in the web.php routes.  Does a limiter need to be added for them, too?

Reply:
Good catch, I hadn't considered auth. What should the limits be for the web group?

I like the idea of environment variables from the other comment. What do you think of:

- RATE_LIMIT_WEB